src/stories/PublicationItemPlatformIcon.js

```diff
@@ -0,0 +1,77 @@
+import {withReadme, withDocs, doc} from 'storybook-readme';
+import {storiesOf} from '@storybook/vue';
+import {object, withKnobs, number, text, select} from '@storybook/addon-knobs';
+
+import PublicationItemPlatformIcon from '../components/PublicationItemPlatformIcon.vue';
+
+storiesOf('PublicationItemPlatformIcon', module)
+    .addDecorator(withKnobs)
+    .addDecorator(withDocs(PublicationItemPlatformIcon.options.__docs))
+    .add('Just Icon', () => {
+        return {
+            components: {PublicationItemPlatformIcon},
+            props: {
+                image: {
+                    default: text('image', 'https://avatars2.githubusercontent.com/u/19777948?s=460&v=4')
+                },
+                socialType: {
+                    default: select('socialType',
+                        [
+                            'twitter',
+                            'vk',
+                            'instagram',
+                            'telegram',
+                            'facebook',
+                        ],
+                        'twitter')
+                },
+            },
+            template: `<publication-item-platform-icon
+                            :image="image" :socialType="socialType"
```

Comment:
Здесь и ниже нужно использовать kebab-case для пропсов:
```suggestion
                            :image="image" :social-type="socialType"
```